src/App.js

```diff
@@ -1,24 +1,22 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
+
+import TaskAdder from './Components/TaskAdder';
+import TaskEditor from './Components/TaskEditor';
+
 import './App.css';
 
 function App() {
+  const [taskList, setTaskList] = useState([]);
+  const [task, setTask] = useState('');
+  const [taskToBeEdited, setTaskToBeEdited] = useState({});
+  const [newDescription, setNewDescription] = useState('');
+  
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <h1 className="heading heading--main">To Do App</h1>
+      <TaskAdder taskList={taskList} setTaskList={setTaskList} task={task} setTask={setTask} taskToBeEdited={taskToBeEdited} setTaskToBeEdited={setTaskToBeEdited} newDescription={newDescription} setNewDescription={setNewDescription}/>
+      <TaskEditor setTaskList={setTaskList} tasks={taskList} taskToBeEdited={taskToBeEdited} newDescription={newDescription} setNewDescription={setNewDescription}/>
```

Comment:
When you have multiple props for a component list them on separate lines to make it more readable.